src/Siganberg.SerilogElasticSearch/Formatter/ElasticSearchFormatter.cs

```diff
@@ -25,7 +25,9 @@ public class ElasticSearchFormatter : ITextFormatter
             {"Elapsed", "durationMs"},
             {"RequestHeaders", "requestHeaders"},
             {"RequestBody", "requestBody"},
-            {"ResponseBody", "responseBody"}
+            {"ResponseBody", "responseBody"},
```

Comment:
I also wanted to see the values of the `contentType` and `contentLength` which can be useful for trouble-shooting.

Reply:
Can you verify from other applications that these 2 are indexed (included) in Kibana?